Configurations/10-main.conf

```diff
@@ -622,9 +630,9 @@ my %targets = (
     },
     "tru64-alpha-cc" => {
         inherit_from     => [ "BASE_unix", asm("alpha_asm") ],
-        cc               => "cc",
-        cflags           => combine("-std1 -tune host -fast -readonly_strings",
-                                    threads("-pthread")),
+        CC               => "cc",
+        cflags           => add("-std1 -tune host -fast -readonly_strings",
```

Comment:
-tune host -fast is an optimization option. Since you keep -O3 as overrideable, this one should be be as well.

Reply:
Cool, will do.

(this is precisely why I asked you to review...  you know most of these flags a lot better than I do)